KNOWN_ISSUES.md

```diff
@@ -3,49 +3,127 @@
 ## Table of Contents
 
 <!-- toc -->
-- [Missing metrics](#missing-metrics)
+- [Kubelet doesn't report metrics for all or subset of nodes](#kubelet-doesnt-report-metrics-for-all-or-subset-of-nodes)
+- [Kubelet doesn't report pod metrics](#kubelet-doesnt-report-pod-metrics)
+- [HPA is unable to get resource utilization](#hpa-is-unable-to-get-resource-utilization)
 - [Incorrectly configured front-proxy certificate](#incorrectly-configured-front-proxy-certificate)
 - [Network problem when connecting with Kubelet](#network-problem-when-connecting-with-kubelet)
+- [Unable to work properly in Amazon EKS](#unable-to-work-properly-in-amazon-eks)
 <!-- /toc -->
 
-## Missing metrics 
+## Kubelet doesn't report metrics for all or subset of nodes
 
 **Symptoms**
 
-Running
+If you run `kubectl top nodes` and not get metrics for all nods in the clusters or some nodes will return `<unknown>`  value. For example:
 ```
-kubectl top pods
-```
-will return empty result for pods older then 1 minute. e.g.
-```
-W1106 20:50:15.238493 73592 top_pod.go:265] Metrics not available for pod default/example, age: 22h29m11.238478174s
-error: Metrics not available for pod default/example, age: 22h29m11.238478174s
+$ kubectl top nodes
+NAME      CPU(cores)  CPU%       MEMORY(bytes)  MEMORY%
+master-1  192m        2%         10874Mi        68%
+node-1    582m        7%         9792Mi         61%
+node-2    <unknown>   <unknown>  <unknown>      <unknown>
 ```
 
 **Debugging**
 
-Please check if your Kubelet is correctly returning pod metrics. You can do that by checking Summary API on any node in your cluster:
+Please check if your Kubelet returns correct node metrics.
+
+You can do that by checking resource metrics on nodes that are missing metrics.
+
 ```console
 NODE_NAME=<Name of node in your cluster>
-kubectl get --raw /api/v1/nodes/$NODE_NAME/proxy/stats/summary
+kubectl get --raw /api/v1/nodes/$NODE_NAME/proxy/metrics/resource
```

Comment:
/cc @serathius I found this command changed, so I made changes to known issues, is this necessary?

Reply:
Let's just make the master branch documentation also mention alternative commands for v0.5 release.

Reply:
Already mentioned in the master branch.
But I don't know if the description is not clear enough or something else.
Some users get confused.
E.g:
https://github.com/kubernetes-sigs/metrics-server/issues/1046

Description of the master branch:
https://github.com/kubernetes-sigs/metrics-server/blob/878da752c36401606cb6f1ed696cb84393e0c5d4/KNOWN_ISSUES.md?plain=1#L31
https://github.com/kubernetes-sigs/metrics-server/blob/878da752c36401606cb6f1ed696cb84393e0c5d4/KNOWN_ISSUES.md?plain=1#L67